src/components/structures/auth/Registration.tsx

```diff
@@ -614,8 +614,8 @@ export default class Registration extends React.Component<IProps, IState> {
                     onServerConfigChange={this.state.doingUIAuth ? undefined : this.props.onServerConfigChange}
                 />
                 { this.renderRegisterComponent() }
-                { goBack }
-                { signIn }
+                <div className="mx_AuthBody_link_goBack">{ goBack }</div>
```

Comment:
The margins of `mx_Dialog_buttons_items` will still be applied even when `goBack` is falsy, creating a weird gap in the UI when we don't render a go back button.

Reply:
Addressed: https://github.com/matrix-org/matrix-react-sdk/pull/8208/commits/dbdf83d3c2dc6ae0f0b7c8decda78542f480569d

Reply:
I guess you are talking about #8160?